tests/utils_tests/test_autoreload.py

```diff
@@ -410,10 +410,11 @@ def patch_autoreload(self, argv):
         return mock_call
 
     def test_manage_py(self):
-        script = Path('manage.py')
+        temp_dir = tempfile.TemporaryDirectory()
+        self.addCleanup(temp_dir.cleanup)
```

Comment:
`tempfile.TemporaryDirectory()` can be used as a context manager:

```suggestion
        with tempfile.TemporaryDirectory() as temp_dir:
```

Reply:
Honestly I'm not a fan of this change. It can indeed be used as a context manager but the indent impacts readability.  However this test is small enough that it doesn't necessarily matter. so I'll leave it up to @felixxm to decide 💪

Reply:
Okay then, no worries. As context managers are a regular, routine Python idiom, even outside tests, I find them quite readable (even with the indentation) and I find it makes the clenaup extremely explicit. Both forms work, though.

Reply:
Thanks both, I changed to a context manager.